src/javaRosa/util.js

```diff
@@ -281,6 +281,11 @@ define([
                     return $this[0].outerHTML;
                 });
                 text = xml.normalize(text.html());
+                if (escape) {
+                    text = text.replace(/(<)|>/g, function (match, lt) {
+                        return lt ? "&lt;" : "&gt;";
+                    });
```

Comment:
`&` chars have already been converted to `&amp;` above on [line 270](https://github.com/dimagi/Vellum/pull/761/files#diff-16525243359147bd8661fbb97da09528R270)